### PR TITLE
create-app: add app-backend plugin to template

### PR DESCRIPTION
**Pull request by Rugvip**
## Hey, I just made a Pull Request!

Not actually verified yet, but up for some early review :grin:

Some more context:

With #3264 now merged, any config loaded by the backend will also be filtered by visibility and injected into the frontend by the app-backend plugins. With that the plugin is a lot less confusing to use and much more powerful than other deployment alternatives with separate app/backends. It's still perhaps not what we'd recommend for a scaled deployment with high availability, but it's gonna be good enough for 95% of all usage and what I think we should recommend going forward.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: .changeset/add-app-backend.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
```
Comment (freben) [inline .changeset/add-app-backend.md:1]: great first example of a create-app changeset!

File: .changeset/add-app-backend.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Add `app-backend` as a backend plugin, and make a single docker build of the backend the default way to deploy backstage.
+
+## Template changes
+
+As a part of installing the `app-backend` plugin, the below changes where made. The changes are grouped into two steps, installing the plugin, and updating the Docker build and configuration.
+
+### Installing the `app-backend` plugin in the backend
+
+First, install the `@backstage/plugin-app-backend` plugin package in your backend. These changes where made for `v0.3.0` of the plugin, and the installation process might change in the future. Run the following from the root of the repo:
```
Comment (freben) [inline .changeset/add-app-backend.md:13]:
```suggestion
First, install the `@backstage/plugin-app-backend` plugin package in your backend. These changes were made for version `0.3.0` of the plugin, and the installation process might change in the future. Run the following from the root of the repo:
```

**Review by freben — APPROVED** [review]
So the time has come

**Comment by benjdlambert** [comment]
Not sure the best place to document this, or if it's documented already, but we should mention that this route will only serve the **production** / **built** version of the frontend, so it does not replace the `yarn start` in the frontend package for development

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**